complete/pom.xml

```diff
@@ -23,20 +23,18 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
-		<dependency>
```

Comment:
It looks like two `spring-boot-starter-test`s snuck in here.

Reply:
Good catch.